examples/pachyderm-cd4ml/index.ipynb

```diff
@@ -0,0 +1,1088 @@
+    "    }\n",
+    "  }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pachctl create pipeline -f deploy.json"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME         VERSION INPUT          CREATED            STATE / LAST JOB  DESCRIPTION                                                                    \n",
+      "rsi_deploy   1       rsi_training:/ 43 seconds ago     \u001b[32mrunning\u001b[0m / \u001b[32msuccess\u001b[0m A pipeline deploys a seldon core model using the most recent model parameters. \n",
+      "rsi_training 1       rsi_data:/     About a minute ago \u001b[32mrunning\u001b[0m / \u001b[32msuccess\u001b[0m A pipeline that trains a statsmodels model based on the provided input data.   \n",
+      "ID                               PIPELINE     STARTED        DURATION  RESTART PROGRESS  DL       UL      STATE   \n",
+      "f5bb1b2009e0452b806681b7e44e403d rsi_deploy   39 seconds ago 2 seconds 0       1 + 0 / 1 15.9KiB  0B      \u001b[32msuccess\u001b[0m \n",
+      "5a45b36d40b74b49832d052b5eae4fe9 rsi_training 53 seconds ago 1 second  0       1 + 0 / 1 1.173KiB 15.9KiB \u001b[32msuccess\u001b[0m \n"
+     ]
+    }
+   ],
+   "source": [
+    "!pachctl list pipeline\n",
+    "!pachctl list job"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. Call the Initial Model to Predict the Future (pre-COVID)\n",
+    "\n",
+    "By now you should have a fully operational Seldon Deployment, driven by your initial data commit. But it's worth checking. Often there are resource limit issues or silly things like that. If you have any issues, fix them first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME                                                    READY   STATUS    RESTARTS   AGE\r\n",
+      "rsi-predictor-default-0-rsi-predictor-c4d7dfbf6-vm56d   2/2     Running   0          41s\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!kubectl -n seldon get po"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now it's time to take a look at the data. In this notebook I want to actually show you the data, so I'm going to parse it into a pandas array. Below I create two helper methods to do this.\n",
+    "\n",
+    "Make sure you've port-forwarded the ingress controller service, or replace localhost with something routable from your machine. For exmaple:\n",
+    "\n",
+    "```sh\n",
+    "kubectl port-forward $(kubectl get pods -n seldon-system -l app.kubernetes.io/name=ambassador -o jsonpath='{.items[0].metadata.name}') -n seldon-system 8080:8080\n",
```

Comment:
We usually recommend to forward it to port `8003`, as per SC configuration page linked at the top of the notebook:
```
!kubectl port-forward $(kubectl get pods -n seldon-system -l app.kubernetes.io/name=ambassador -o jsonpath='{.items[0].metadata.name}') -n seldon-system 8003:8080
```

(just making a note due to consistency with other exampels as users running it probably already have 8003 port forwarded in separate terminal)